tests/features/SearchableTest.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+use Faker\Factory;
+use Tests\TestCase;
+use LaravelEnso\Core\app\Models\User;
+use Illuminate\Database\Eloquent\Model;
+use LaravelEnso\Searchable\app\Classes\Finder;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use LaravelEnso\PermissionManager\app\Models\Permission;
+
+class SearchableTest extends TestCase
+{
+    use  RefreshDatabase;
+
+    private $testModel;
+    private $query;
```

Comment:
what's this for?

Reply:
This variable is indeed unused. I have addressed the issue in my latest commit.